app/scripts/services/metrics.js

```diff
@@ -50,7 +51,12 @@ angular.module("openshiftConsole")
 
     function getRequestURL(config) {
       return getMetricsURL().then(function(metricsURL) {
-        var template = metricsURL + POD_GAUGE_TEMPLATE;
+        var template;
+        if (config.type !== undefined && config.type === "counter") {
```

Comment:
Maybe just

```js
if (config.type === 'counter') { ... }
```

although probably worth a comment saying gauge is assumed when type is not set